pkg/util/pki/certificatetemplate.go

```diff
@@ -377,38 +377,3 @@ func CertificateTemplateFromCertificateSigningRequest(csr *certificatesv1.Certif
 		CertificateTemplateValidateAndOverrideKeyUsages(ku, eku),          // Override the key usages, but make sure they match the usages in the CSR if present
 	)
 }
-
```

Comment:
These functions were deprecated since `v1.13.0` which was released ~4 months ago on Sep 12, 2023.
 * https://github.com/cert-manager/cert-manager/pull/6019

I think that's probably enough notice and in the README file we say:

https://github.com/cert-manager/cert-manager/blob/b656118beefd80cc9dfb9892387caab96ef54f87/README.md?plain=1#L84-L86